Competition2017/src/competition/operator_interface/OperatorCommandMap.java

```diff
@@ -17,4 +22,16 @@ public void setupMyCommands(
         operatorInterface.leftButtons.getifAvailable(1).whenPressed(myCommand);
     }
     */
+    
+    @Inject
+    public void setupCollectorCommands(
+            OperatorInterface oi,
+            EjectCollectorCommand eject,
+            IntakeCollectorCommand intake,
+            StopCollectorCommand stop
+    )   {
+        oi.leftButtons.getifAvailable(1).whileHeld(eject);
+        oi.leftButtons.getifAvailable(2).whileHeld(intake);
+        oi.rightButtons.getifAvailable(1).whenPressed(stop);
```

Comment:
Since the stop command is the default, it seems like we shouldn't need a separate "stop" button.